### ISSUE TITLE
Omega Mute Soundmap Text File Restoration

### ISSUE THREAD

**Issue by SpookyTheOoze** [ricksouth/serilum-mc-mods#2640]
## **Information**
Minecraft version: 1.21
Modloader: Fabric
Fabric loader version: 0.16.4
Environment: Multiplayer

Mod name: Omega Mute
Mod version: 1.21.1-4.2

## **Description**
I accidentally applied the Serilum settings in-game after installing the mod initially, through the CurseForge application, and before having read the entire mod description assumed I had to wipe the contents of the soundmap.txt file to reset it. So I saved, closed the file, and re-booted Minecraft. Eventually I tried to remake it by making my own soundmap.txt file and pasting in all the sound strings in the game and have now arrived at the crossroads of the mod working in-game, making the appropriate changes in the document, but the sounds in-game not getting muted.

**Comment by ricksouth** [comment]
That should've reset it. You can also run

/omegamute unmute .

To reset everything

**Comment by SpookyTheOoze** [comment]
I tried that too, and then tried re-muting the sounds I wished to mute, and still nothing.

**Comment by ricksouth** [comment]
Are the sounds muted in /omegamute listen?

**Comment by SpookyTheOoze** [comment]
Nope. The sounds I wanted to mute, those related to the Nether Portal, are still playing in /omegamute listen. "block.portal.ambient", "block.portal.trigger", and "block.portal.travel" all still occur.

**Comment by ricksouth** [comment]
Hmm that's strange. Can you close the game, delete the entire mute file, start the game, and then try to mute them again?

Otherwise I'll have to take a look at what the issue is. Apologies for the inconvenience.

**Comment by SpookyTheOoze** [comment]
Alright I closed the game, deleted the mod and entire omegamute folder, then reinstalled the mod, remade the folder and the text file inside, then pasted all the sound strings back in again, saved, closed it all, then booted up the game. The sounds did not mute right away as I stood next to the portal, it only seemed to take effect as I went through it. 

And this even working is something I find strange, considering I DID THIS EXACT THING EARLIER AND IT DID NOTHING.

**Comment by James103** [comment]
Can you please try attaching the Omega Mute config file containing the list of sounds to mute?
